### DISCUSSION TITLE
Spawning language server with cmd: `lua-language-server` failed.

### DISCUSSION THREAD

**Discussion by GershomVaiman** [NvChad/NvChad#2887] [Q&A] [answered]
Hello! After installing NvChad on my Neovim, I see this error. If I try to open init.lua, I see the same error.

![image](https://github.com/NvChad/NvChad/assets/105606049/ed3c6274-db29-490e-ab0c-c15268c5f548)

**Comment by siduck** [comment, marked as answer]
masoninstallall, run that command

**Comment by GershomVaiman** [comment]
![image](https://github.com/NvChad/NvChad/assets/105606049/869a2310-008b-4656-ae11-e91b3e8fe51c)

I did this, but the error is still there

![image](https://github.com/NvChad/NvChad/assets/105606049/e89df4a8-ff29-4642-b518-37d7ce262fdf)

**Reply by siduck** [reply]
@GershomVaiman  run lua-language-server in neovim's terminal and show

I hope you are not using that NIXOS, guix, gobolinux, haikuOS, serenityOS :joy:
